adapters/nextmillennium/nextmillennium.go

```diff
@@ -0,0 +1,150 @@
+package nextmillennium
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+type NextMillenniumBidRequest struct {
+	ID   string `json:"id"`
+	Test uint8  `json:"test,omitempty"`
+	Ext  struct {
+		Prebid struct {
+			StoredRequest struct {
+				ID string `json:"id"`
+			} `json:"storedrequest"`
+		} `json:"prebid"`
+	} `json:"ext"`
+}
+
+//MakeRequests prepares request information for prebid-server core
+func (adapter *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	resImps, err := getImpressionsInfo(request.Imp)
+	if len(err) > 0 {
+		return nil, err
+	}
+
+	result := make([]*adapters.RequestData, 0, len(resImps))
+	for _, imp := range resImps {
+		bidRequest, err := adapter.buildAdapterRequest(request, imp)
+		if err != nil {
+			return nil, []error{err}
+		} else {
+			result = append(result, bidRequest)
+		}
```

Comment:
Nitpick: There is no need for `else` statement when the `if` statement returns. Can be simplified to:
```
if err != nil {
  return nil, []error{err}
}
result = append(result, bidRequest)
```